src/open_samus_returns_rando/files/schema.json

```diff
@@ -497,6 +497,17 @@
                             }
                         }
                     }
+                },
+                "music_shuffle_dict": {
+                    "description": "A dictionary mapping music tracks to the output of music mode.",
```

Comment:
I don't get what this has to do with `music mode`. 
It swaps one track with the other.

Reply:
I don't understand the confusion.

Reply:
Music mode has no effect here and is completely irrelevant. A more accurate description would be something along the lines of "A dictionary mapping, where the key is a track that gets swapped with the value"

Reply:
Fwiw this was made before the rdv pr, so this is just a small wording error on my part. I'll fix it.